docs/user_guide/assessment.rst

```diff
@@ -704,10 +604,82 @@ The names of the generated functions are of the form
 For example :code:`fairlearn.metrics.accuracy_score_difference` and
 :code:`fairlearn.metrics.precision_score_group_min`.
 
-.. _control_features_metrics:
 
-Control features for grouped metrics
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Intersecting Groups
+-------------------
+
+The :class:`MetricFrame` class supports intersectional fairness in two ways:
+multiple sensitive features, and control features.
+Both of these can be used simultaneously.
+One important point to bear in mind when performing an intersectional analysis
+is that some of the intersections may have very few members (or even be empty).
+This can affect the confidence interval associated with the computed metrics.
```

Comment:
Suggest adding a little more detail about how the confidence interval changes when intersectional groups are small or empty.

Reply:
Done.